realm/realm-jni/src/util.cpp

```diff
@@ -185,21 +185,6 @@ jclass GetClass(JNIEnv* env, const char* classStr)
     return myClass;
 }
 
-void jprint(JNIEnv *env, char *txt)
```

Comment:
no idea, didn't spend time to check. No one is calling them so instead of making them `static`, I just removed them. They would be easily back by checking git history if needed.